### PR TITLE
NPE on Collections.unmodifiableList

### PR DESCRIPTION
**Pull request by ciprimare**
Fixes issue #33

### CODE REVIEW

File: feature-adapter/src/main/java/com/groupon/featureadapter/FeaturesAdapter.java [hunk 1]
```diff
@@ -123,7 +123,10 @@ public void onViewRecycled(ViewHolder holder) {
   @SuppressWarnings("WeakerAccess")
   public void updateFeatureItems(MODEL model) {
     for (FeatureController<MODEL> featureController : featureItems.getFeatureControllers()) {
-      dispatchFeatureUpdate(toFeatureUpdate(featureController, model));
+        final FeatureUpdate featureUpdate = toFeatureUpdate(featureController, model);
+        if (featureUpdate.newItems != null && featureUpdate.diffResult != null) {
```
Comment (stephanenicolas) [inline feature-adapter/src/main/java/com/groupon/featureadapter/FeaturesAdapter.java]: testing newItems should be enough. Though this test is right too.

Comment (stephanenicolas) [reply feature-adapter/src/main/java/com/groupon/featureadapter/FeaturesAdapter.java]: Actually, no, keep it, we have the same test on the Rx side :)

**Comment by stephanenicolas** [comment]
RTM after testing in MGA

File: feature-adapter/src/main/java/com/groupon/featureadapter/FeaturesAdapter.java (hunk 1 above)
Comment (ciprimare) [reply feature-adapter/src/main/java/com/groupon/featureadapter/FeaturesAdapter.java]: yes the condition is exactly the same with the one used on the Rx side :)

**Pull request merged**